libs/core/src/components/sage-input/sage-input.tsx

```diff
@@ -0,0 +1,119 @@
+import { Component, Event, EventEmitter, h, Host, Prop } from '@stencil/core';
+
+/**
+ * @slot - Content is placed between the opening closing tags
+ */
+@Component({
+  tag: 'sage-input',
+  styleUrl: 'sage-input.scss',
+  shadow: true,
+})
+export class SageInput {
+  /**
+   * Indicates whether or not the input field is disabled
+   */
+  @Prop() disabled?: boolean;
+
+  /**
+   * Specifies the error text and provides an error-themed treatment to the field
+   */
+  @Prop() errorText?: string;
+
+  /**
+   * Displays a hint or description of the input field
+   */
+  @Prop() hint?: string;
+
+  /**
+   * Indicates whether or not the input field is invalid or throws an error
+   */
+  @Prop() invalid?: boolean;
+
+  /**
+   * A unique identifier for the input field
+   */
+  @Prop() inputId: string;
+
+  /**
+   * Text to be displayed as the form label
+   */
+  @Prop() label?: string;
+
+  /**
+   * Specifies the name. Submitted with the form name/value pair
+   */
+  @Prop() name?: string;
+
+  /**
+   * Specifies a short hint that describes the expected value of the input field
+   */
+  @Prop() placeholder?: string;
+
+  /**
+   * Indicates whether or not the input field is readonly
+   */
+  @Prop() readonly?: boolean;
+
+  /**
+   * Indicates whether or not the input field is required
+   */
+  @Prop() required?: boolean;
```

Comment:
I'm not seeing the required toggle in storybook working

Reply:
I'm still not seeing `required="required"` being set on the `input` element when I toggle this on in storybook